### DISCUSSION TITLE
Avalonia Notification Service

### DISCUSSION THREAD

**Discussion by DamianSuess** [AvaloniaCommunity/Prism.Avalonia#27] [Ideas]
Should we implement support for Avalonia's pop-up notifications (_`INotificationService`_) or should it be housed in its own repository such as, `Prism.Avalonia.Extensions`?

Notification cards were implemented in [Avalonia PR #3389](https://github.com/AvaloniaUI/Avalonia/pull/3389). As an example, check out [Prism.Avalonia Outlookish](https://github.com/DamianSuess/Learn.PrismAvaloniaOutlookish).

Just a thought.

**Comment by OmidID** [comment]
Hello @DamianSuess
I suggesting to create another package as you said Prism.Avalonia.Extensions
And I see that PR is not finalized.

**Reply by DamianSuess** [reply]
I'm with you, @OmidID. This would allow for extensibility and flexibility without affecting the core library. 
Thanks for the feedback 👍  Hope you're doing well, sir, and work is allowing you time to relax during your off-time.

**Reply by OmidID** [reply]
Thanks a lot :) 
And great job 👍  you are doing amazing.
